specification/datalake-analytics/data-plane/Microsoft.DataLakeAnalytics/stable/2016-11-01/catalog.json

```diff
@@ -2788,23 +2781,20 @@
         ],
         "responses": {
           "200": {
-            "description": "Successfully granted the access control list (ACL) entry to the specified database."
-          },
-          "default": {
-            "description": "404 - Not found."
+            "description": "Successfully granted the access control list (ACL) entry to the Data Lake Analytics catalog."
```

Comment:
Why we are removing the `default` from responses section?

Reply:
@anuchandy This was to follow the the rest of the APIs in that file.  These ACL APIs were the first ones that I added to this Swagger file.  But now as I look at them, I see that `default` is not really needed (or is redundant) because if it's not 200, we throw with the status code.